pkg/taskrunner/tasks_test.go

```diff
@@ -0,0 +1,135 @@
+package taskrunner_test
+
+import (
+	"archive/zip"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/konstructio/kubefirst-api/pkg/taskrunner"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExec(t *testing.T) {
+	zerolog.SetGlobalLevel(zerolog.Disabled)
+
+	dir, err := os.Getwd()
+	assert.NoError(t, err)
```

Comment:
If the directory you need is relative to where this file is, you can use relative routes without having to rely on `os.Getwd`.

Also, thanks for adding `testify`! Let me add you one more:

On testify, sometimes you want to stop the test instead of asserting but continuing. If that's the case, you can swap `assert.NoError` for `require.NoError` to achieve that goal. 99% of the time we do want to stop processing instead of continuing.